resources/assets/js/components/ApplicationReview/ApplicationTimelineReviewRoot.tsx

```diff
@@ -0,0 +1,165 @@
+/* eslint-disable camelcase */
+import * as React from "react";
+import ReactDOM from "react-dom";
+import { useIntl } from "react-intl";
+import { useDispatch } from "react-redux";
+import { DispatchType } from "../../configureStore";
+import { getLocale } from "../../helpers/localize";
+import {
+  useCriteria,
+  useExperiences,
+  useExperienceSkills,
+  useFetchReviewApplicationData,
+  useJob,
+  useJobApplicationAnswers,
+  useJobPosterQuestions,
+  useReviewedApplication,
+  useSkills,
+  useUser,
+} from "../../hooks/applicationHooks";
+import { Portal } from "../../models/app";
+import { ReviewStatus } from "../../models/types";
+import { loadingMessages } from "../Application/applicationMessages";
+import ApplicationPreview from "../Application/Review/ApplicationPreview";
+import RootContainer from "../RootContainer";
+import ApplicationReviewNav from "./ApplicationReviewRoot";
+
+interface ApplicationTimelineReviewRootProps {
+  applicationId: number;
+  applicantUserId: number;
+  jobId: number;
+  reviewStatuses: ReviewStatus[];
+  portal: Portal;
+}
+
+const ApplicationTimelineReviewRoot: React.FunctionComponent<ApplicationTimelineReviewRootProps> = ({
+  applicationId,
+  applicantUserId,
+  jobId,
+  portal,
+  reviewStatuses,
+}) => {
+  const intl = useIntl();
+  const locale = getLocale(intl.locale);
+  const dispatch = useDispatch<DispatchType>();
+
+  const {
+    experiencesLoaded,
+    experienceConstantsLoaded,
+    skillsLoaded,
+  } = useFetchReviewApplicationData(
+    applicantUserId,
+    applicationId,
+    jobId,
+    dispatch,
+  );
+
+  const application = useReviewedApplication(applicationId);
+  const applicantUser = useUser(applicantUserId);
```

Comment:
You can use useApplicationUser instead, because right now, the Application API returns the whole Applicant and User along with it, which means we don't need to make a seperate request for User.